std/csv.d

```diff
@@ -52,10 +51,8 @@
  }
  * }
  * -------
- *
```

Comment:
I started to fix that, but realized it's not a good idea and should better be fixed directly in Ddoc. After a unittest there should be one empty line allowed without inserting any new line.